delta-lake/delta-20x/src/main/scala/org/apache/spark/sql/delta/rapids/delta20x/Delta20xRuntimeShim.scala

```diff
@@ -47,4 +48,7 @@ class Delta20xRuntimeShim extends DeltaRuntimeShim {
   override def unsafeVolatileSnapshotFromLog(deltaLog: DeltaLog): Snapshot = {
     deltaLog.snapshot
   }
+
+  override def fileFormatFromLog(deltaLog: DeltaLog): FileFormat =
+    deltaLog.fileFormat(deltaLog.snapshot.metadata)
```

Comment:
Curious why this isn't `deltaLog.fileFormat()` to match the original code and what is being used for delta21x version?  Seems like delta20x, delta21x, and delta22x versions should all be consistent here either way.

Reply:
This was unintentional. I have made these consistent now.